### PR TITLE
Add docs for new distance ellipsoid

### PR DESCRIPTION
**Pull request by graetzer**

### CODE REVIEW

File: 3.5/aql/functions-geo.md [hunk 1]
```diff
@@ -58,16 +58,18 @@ This function can be **optimized** by a S2 based [geospatial index](../indexing-
...
+`GEO_DISTANCE(geoJsonA, geoJsonB, ellipsoid) → distance`
 
 Return the distance between two GeoJSON objects, measured from the **centroid**
 of each shape. For a list of supported types see the
 [geo index page](../indexing-geo.html#geojson).
 
 - **geoJsonA** (object): first GeoJSON object
 - **geoJsonB** (object): second GeoJSON object
+- **ellipsoid** (string, *optional*): reference ellipsoid to use.
+  Supported are `"sphere"` and `"wgs84"`.
```
Comment (Simran-B) [inline 3.5/aql/functions-geo.md]: @graetzer Is `"sphere` the default?

Comment (graetzer) [reply 3.5/aql/functions-geo.md]: yes

Comment (Simran-B) [inline 3.5/aql/functions-geo.md]:
```suggestion
  Supported are `"sphere"` (default) and `"wgs84"`.
```

File: 3.5/aql/functions-geo.md [hunk 2]
```diff
@@ -81,6 +81,31 @@ FOR doc IN collectionName
...
+
+```js
+LET polygon = {
+type: "Polygon",
+coordinates: [[[-11.5, 23.5], [-10.5, 26.1], [-11.2, 27.1], [-11.5, 23.5]]]
+}
+RETURN GEO_AREA(polygon, "wgs84")
+```
+
+
```
Comment (Simran-B) [inline 3.5/aql/functions-geo.md]: @graetzer Will this be backported to v3.5?

Comment (graetzer) [reply 3.5/aql/functions-geo.md]: good point, I have to ask @joerg84 / @ifcologne

Comment (joerg84) [reply 3.5/aql/functions-geo.md]: IMO this should be part of 3.5.1 unless any objections...

Comment (jsteemann) [reply 3.5/aql/functions-geo.md]: Functionality has been added to 3.5.1 already by @graetzer.

**Pull request merged**